test/integration/IntegrationTestHelper.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {Test, console, stdError} from "forge-std/Test.sol";
+import {Users} from "test/helpers/Users.sol";
+import {Well, Call, IERC20} from "src/Well.sol";
+import {Auger} from "src/Auger.sol";
+import {Aquifer} from "src/Aquifer.sol";
+import {ConstantProduct2} from "src/functions/ConstantProduct2.sol";
+import {LibContractInfo} from "src/libraries/LibContractInfo.sol";
+import {IWellFunction} from "src/interfaces/IWellFunction.sol";
+
+/// @dev helper struct for quickly loading user / well token balances
+struct Balances {
+    uint[] tokens;
+    uint lp;
+    uint lpSupply;
+}
+
+abstract contract IntegrationTestHelper is Test {
```

Comment:
Is it possible for this to inherit TestHelper and override `setupWell`, `mintTokens` etc.? Many of them are the same.